build/scraper.mjs

```diff
@@ -12,6 +12,7 @@ import VersionScraper from './wikia/scrapers/VersionScraper.mjs';
 import readJson from './readJson.mjs';
 
 import { get, getJSON, retryAttempts } from './network.mjs';
+import ArchwingScraper from './wikia/scrapers/ArchwingScaper.mjs';
```

Comment:
this should be above ModScraper... not sure why this got imported lie this

Reply:
auto import ftw right